### ISSUE TITLE
Composer-less / CLI-less installation archive

### ISSUE THREAD

**Issue by x9t9** [firefly-iii/firefly-iii#1762]
First of all - I must say that this project is one of the best that I have seen when come to financial management .
That being said - 

TL;DR;
I think that providing a simple bundle as a release archive that has all the necessary ingredients for a simple drag-n-drop install would help greatly.
I believe that this could really help the popularity of firefly as it would allow more audiences.

Longer version : 

A scenario where a user just need to "unzip" an archive, edit a simple db/env config file, and then all the rest is handled automatically ( with some input ) would be the preferred method to drop technical barriers.

To be clear - the current installation process is in fact easy - but only if you have some pretty solid level of developer knowledge ( and linux CLI ) AND have all the tools already in place and configured - as  What I refer to is an install process similar to other web based php projects ( most famous example would be wordpress ) where you can ftp the archive to any shared-hosting server and voila ...

I must say that this issue was triggered after I myself tried to install firefly-iii on a two different servers ( both were not mine - one win. one Ubuntu ) and the whole process took me almost 3 hours on each - and the main problem was not even firefly-iii - it was composer which was not originally installed and just kept bugging ( yep - a LOT of people do not use composer - including me ).

Is there a technical reason why NOT to supply such an archive to facilitate installation process for less-savvy users / webmasters / passers-by ?

**Comment by x9t9** [comment]
Wow , nice answer .
So just to be clear , If someone ( e.g. - me ) would make such a package with companion installation scripts ( for "consistently lazy idiots" with "less intelligence" ) - you would still refuse to integrate or publish it ?

**Comment by JC5** [comment]
I’ll happily link to it from the readme but it’s not going to be part of the package.
